azimuth/config.py

```diff
@@ -311,11 +353,40 @@ class SyntaxConfig(CommonFieldsConfig):
     syntax: SyntaxOptions = SyntaxOptions()
 
 
+class LanguageConfig(CommonFieldsConfig):
+    # Language configuration sets multiple config values; see reference dictionary for details
+    language: SupportedLanguage = SupportedLanguage.en
+
+    @root_validator()
+    def dynamic_language_config_values(cls, values):
+        if values["behavioral_testing"]:
+            values["behavioral_testing"].neutral_token.prefix_list = (
+                values["behavioral_testing"].neutral_token.prefix_list
+                or config_defaults_per_language[values["language"]].prefix_list
+            )
+            values["behavioral_testing"].neutral_token.suffix_list = (
+                values["behavioral_testing"].neutral_token.suffix_list
+                or config_defaults_per_language[values["language"]].suffix_list
+            )
+        values["syntax"].spacy_model = (
+            values["syntax"].spacy_model
+            or config_defaults_per_language[values["language"]].spacy_model
+        )
+        values["syntax"].subj_tags = (
+            values["syntax"].subj_tags or config_defaults_per_language[values["language"]].subj_tags
+        )
+        values["syntax"].obj_tags = (
+            values["syntax"].obj_tags or config_defaults_per_language[values["language"]].obj_tags
+        )
```

Comment:
Detail, but with two variables, we can make the code a lot cleaner, with less repetition of the error-prone strings.
```suggestion
        defaults = config_defaults_per_language[values["language"]]
        if values["behavioral_testing"]:
            neutral_token = values["behavioral_testing"].neutral_token
            neutral_token.prefix_list = neutral_token.prefix_list or defaults.prefix_list
            neutral_token.suffix_list = neutral_token.suffix_list or defaults.suffix_list
        values["syntax"].spacy_model = values["syntax"].spacy_model or defaults.spacy_model
        values["syntax"].subj_tags = values["syntax"].subj_tags or defaults.subj_tags
        values["syntax"].obj_tags = values["syntax"].obj_tags or defaults.obj_tags
```

Reply:
Yes, great! Thanks. Did the same with `values["syntax"]` for kicks and also consistency. Plus it gets rid of a few lines of code because it makes things fit on one line.